src/utils/static-eval.ts

```diff
@@ -327,7 +327,7 @@ const visitors: Record<
 
     return {
       test: node.test,
-      then: thenValue.value,
+      then: thenValue.value === URL ? undefined : thenValue.value,
```

Comment:
This looks wrong. Why would URL always be undefined? 

I suspect the root cause is somewhere else since this code represents the ternary from what I can tell.